ca_description/urdf/sensors/tmp.urdf

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" ?>
```

Comment:
How are you adding files to the stash? I recommend you to use:
```
git add file_name1 file_name2
git commit -m "message"
```

With this approach, you will avoid pushing code like this one.

Reply:
Will use this from now on.